ReduxSimple/ReduxStore.cs

```diff
@@ -46,5 +43,20 @@ public IObservable<object> ObserveAction()
         {
             return _actionSubject.OfType<T>().AsObservable();
         }
+
+        private void GoToState(TState state, object action)
+        {
+            State = state;
+
+            if (action != null)
```

Comment:
This is breaking change compared to the last version. Let's create an issue to discuss how to handle null action and remove the check on this PR.

Reply:
I created #10 to discuss it.

Reply:
The original intent of this check was to allow the overloaded method to pass a `null` in here, but you have a good point about breaking change. I'll revisit this.

Reply:
Well. Let's see if you still need to do it on your next iteration.